manim/mobject/three_dimensions.py

```diff
@@ -732,6 +791,71 @@ def get_start(self):
     def get_end(self):
         return self.end
 
+    @staticmethod
+    def perp_to_line(point, line, **kwargs):
```

Comment:
An implementation with a `classmethod` would be more appropriate here. Plus, being able to control the length of the line would be nice.